packages/pds/src/db/migrations/20230505T201944381Z-mute-lists.ts

```diff
@@ -0,0 +1,71 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('list')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('name', 'varchar', (col) => col.notNull())
+    .addColumn('purpose', 'varchar', (col) => col.notNull())
+    .addColumn('description', 'varchar')
+    .addColumn('descriptionFacets', 'varchar')
+    .addColumn('avatarCid', 'varchar')
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .execute()
+
+  await db.schema
+    .createIndex('list_creator_idx')
+    .on('list')
+    .column('creator')
+    .execute()
+
+  await db.schema
+    .createTable('list_item')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('subjectDid', 'varchar', (col) => col.notNull())
+    .addColumn('listUri', 'varchar', (col) => col.notNull())
+    .addColumn('reason', 'varchar')
+    .addColumn('reasonFacets', 'varchar')
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addUniqueConstraint('list_item_unique_subect_in_list', [
+      'creator',
+      'listUri',
+      'subjectDid',
+    ])
+    .execute()
+
+  await db.schema
+    .createIndex('list_item_subject_idx')
+    .on('list_item')
+    .column('subjectDid')
+    .execute()
+
+  await db.schema
+    .createTable('list_mute')
+    .addColumn('listUri', 'varchar', (col) => col.notNull())
+    .addColumn('mutedByDid', 'varchar', (col) => col.notNull())
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addPrimaryKeyConstraint('list_mute_pkey', ['mutedByDid', 'listUri'])
+    .execute()
+
+  // missed index in `actor-block-init` migration
+  await db.schema
+    .createIndex('actor_block_creator_idx')
+    .on('actor_block')
+    .column('creator')
+    .execute()
```

Comment:
I think `actor_block`'s unique constraint on `(creator, subjectDid)` may actually be doing the trick for us.

Reply:
oh yup yknow you're totally right

i coulda sworn i checked that 😅